donate/payments/views.py

```diff
@@ -797,7 +801,9 @@ def form_valid(self, form, send_data_to_basket=True):
                     exc_info=True
                 )
 
-        elif send_data_to_basket:
+        if send_data_to_basket:
+            basket.subscribe(data['email'], 'mozilla-foundation', lang=self.request.LANGUAGE_CODE)
+            # TODO: When going live we can remove the SQS commands below
             data['source_url'] = self.request.build_absolute_uri()
             data['lang'] = self.request.LANGUAGE_CODE
             queue.enqueue(send_newsletter_subscription_to_basket, data)
```

Comment:
Note: I've kept SQS in here for the time being so this can be deployed without breaking the current site. We should remove this after the data migration work is complete.

Reply:
Let's separate it, see above. We don't want the same event sent twice as it'll end up in the same place